### DISCUSSION TITLE
Support text description on Threshold lines on TimeSeries Panels

### DISCUSSION THREAD

**Discussion by mourya92** [grafana/grafana#62049] [Dashboards, Panels & Visualizations]
Is it possible to write a text description on Threshold lines indicating what the threshold is about. 

Example:
<img width="791" alt="Screen Shot 2023-01-24 at 8 02 01 PM" src="https://user-images.githubusercontent.com/8276235/214456463-e6f48679-4234-49a6-855c-d06ab441ad60.png">

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82923. Feel free to continue the discussion around this there. Thank you!
